packages/nextjs/components/builders/BuildersGrid.tsx

```diff
@@ -0,0 +1,121 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { normalize } from "viem/ens";
+import { useEnsAvatar, useEnsName } from "wagmi";
+import { useBuildersEvent } from "~~/hooks/useBuildersEvent";
+import { BUILDER_DATA } from "~~/utils/constants";
+
+export function BuildersGrid() {
+  const { events: uniqueEvents, isLoading, error: errorReadingEvents } = useBuildersEvent();
+
+  const getBuilderData = (address: string) => {
+    return BUILDER_DATA[address as keyof typeof BUILDER_DATA];
+  };
```

Comment:
Since this is used only on `BuilderCard` it doesn't actually need to be a function, you can access `BUILDER_DATA` directly just once for each `BuilderCard`  instance.